### ISSUE TITLE
Fix docstrings throughout code

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#80]
The docstrings are not all correctly formatted.  Need to systematically go through the code and fix them. And add/expand them where needed.  

Especially useful in:
- [ ] physicsmodel
- [ ] observationmodel
- [x] fitting

**Comment by petiay** [comment]
Is there a more specific location where the docstrings need attention or are they throughout the code?  Happy to give it a try, and expand them for code I am familar with.

**Comment by karllark** [comment]
Nope.  Just all throughout the code.  You could focus on the code that is covered with the CI tests (use the coverage button on the main page).  That would avoid updating docstrings that have a high probability of being deleted due to never being used.
